packages/@aws-cdk/aws-sns-subscriptions/package.json

```diff
@@ -58,21 +58,6 @@
     "organization": true
   },
   "jest": {
```

Comment:
What's the rationale for leaving an empty `jest` section? It's usually not a good practice to rely on empty objects as values.